sched/wqueue/kwork_queue.c

```diff
@@ -148,16 +107,39 @@ static void work_qqueue(FAR struct kwork_wqueue_s *wqueue,
 int work_queue(int qid, FAR struct work_s *work, worker_t worker,
                FAR void *arg, clock_t delay)
 {
+  irqstate_t flags;
+
+  /* Remove the entry from the timer and work queue. */
+
+  work_cancel(qid, work);
+
+  /* Interrupts are disabled so that this logic can be called from with
+   * task logic or ifrom nterrupt handling logic.
```

Comment:
typo:

`ifrom nterrupt handling logic`

should be

`from interrupt handling logic`

Reply:
> typo:
> 
> `ifrom nterrupt handling logic`
> 
> should be
> 
> `from interrupt handling logic`

Done. Thank you.